### DISCUSSION TITLE
Which new framework?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [aleksey-hoffman/sigma-file-manager#240] [Q&A] [answered]
> [You will be pleased to know that I'm currently rewriting the app on a new framework.](https://github.com/aleksey-hoffman/sigma-file-manager/issues/218#issuecomment-1471535723)

@aleksey-hoffman, which framework is that?

**Comment by aleksey-hoffman** [comment, marked as answer]
The new version is built on Tauri. It has many advantages over Electron. 
So the backend is going to be in Rust, instead of Node.js. On the frontend I'm also upgrading the stack to Vue 3 + Typescript.
